src/components/DailyReportFormUpdate/index.js

```diff
@@ -1,212 +1,262 @@
+                onBlur={this.handleBlur('title')}
+                error={shouldMarkError('title') ? errors.title : false}
+              />
+
+              <RadioInput
+                onRadioChange={this.handleRadioChange}
+                emotion={emotion}
+              />
+
+              <CreateTaskForm
+                tasks={tasks}
+                handleTaskChange={this.handleTaskChange}
+                handleAddNewTask={this.handleAddNewTask}
+                handleRemoveTask={this.handleRemoveTask}
+                errors={errors}
+              />
+
+              <TextArea
+                type="textarea"
+                label="Plan for next day"
+                name="plan"
+                value={plan}
+                onChange={this.handleTextChange}
+                onBlur={this.handleBlur('plan')}
+                error={shouldMarkError('plan') ? errors.plan : false}
               />
-            </CenterWrapper>
-          )}
-        </Mutation>
-      );
-    }}
-  </Query>
-);
+
+              {comment !== '' && comment !== null ? (
+                <TextArea
+                  type="textarea"
+                  label="Comment"
+                  name="comment"
+                  value={comment}
+                  onChange={this.handleTextChange}
+                />
+              ) : (
+                <TextArea
+                  type="textarea"
+                  label="Comment"
+                  name="comment"
+                  value=""
+                  onChange={this.handleTextChange}
+                />
+              )}
+
+              <Button
+                disabled={isDisabled}
+                type="submit"
+                compact
+                color='blue'
+                size='tiny'
+              >
+                Update
+              </Button>
+
+              <Message
+                success
+                header='Update Successfully!'
+                content="Your report has been updated."
+              />
+            </Form>
+          )
+        }}
+      </Mutation>
+    );
+  }
+}
 
 const UPDATE_DAILY_REPORT_QUERY = gql`
   mutation UPDATE_DAILY_REPORT_QUERY(
-  $id: ID!
-  $title: String
-  $emotion: String
-  $achievement: String
-  $plan: String
-  $comment: String
+    $id: ID!
+    $title: String
+    $emotion: String
+    $tasks: [TaskInput!]
+    $plan: String
+    $comment: String
   ) {
     updateDailyReport(
       id: $id
       title: $title
       emotion: $emotion
-      achievement: $achievement
+      tasks: $tasks
       plan: $plan
       comment: $comment
     ) {
       id
       title
       emotion
-      achievement
+      tasks {
+        id
+        project {
+          id
+          title
+        }
+        url
+        logtime
+      }
       plan
       comment
       createdAt
     }
   }
 `;
 
+// Utils function check errors object that passing. Accept errors argument object.
+// return false if not have any error.
+function isErrors(errors) {
```

Comment:
I think there are 2 way to refactor `isErrors` (because it's declared in update form & create form)
1. Extract `isErrors(errors){...}` to `src/utils/isErrors.js` or something and then import it 
or
2. Delete isErrors(errors){...} from `DailyReportFormUpdate` and import it from `DailyReportFormCreate`

Reply:
You're right. I'll fix this.